Database/Corrections/cataQuestFixes.lua

```diff
@@ -3675,6 +3675,9 @@ function CataQuestFixes.Load()
         [26116] = { -- Kinelory Strikes
             [questKeys.triggerEnd] = {"Protect Kinelory", {[zoneIDs.ARATHI_HIGHLANDS] = {{54.85,55.55}}}},
         },
+        [26118] = { -- Seize the Ambassador
+            [questKeys.extraObjectives] = {{nil,Questie.ICON_TYPE_MOUNT_UP,l10n("Arrest Ambassador Slaghammer and bring him to the High Seat"),0,{{"monster",42146}}}},
```

Comment:
Now we lost the first part 😅 

```suggestion
            [questKeys.triggerEnd] = {"Arrest Ambassador Slaghammer and bring him to the High Seat", {[zoneIDs.IRONFORGE] = {{39.65,56.85}}}},
            [questKeys.extraObjectives] = {{nil,Questie.ICON_TYPE_MOUNT_UP,l10n("Arrest Ambassador Slaghammer"),0,{{"monster",42146}}}},
```

We also need to add `"Arrest Ambassador Slaghammer"` to the translations, so we can potentially translate it.

Reply:
https://github.com/Questie/Questie/pull/6218/commits/e78030d3fc36378f5e5a3abbfed7383fc393f7aa